finance-portal/templates/ingress.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  annotations:
+    kubernetes.io/ingress.class: nginx
```

Comment:
Please move annotations to config values.yml.

Ingress template must be generic (and made specific in the values.yml via config). <-- i.e. it will allow any ingress-controller to be used, and thus it will not be specific to the nginx flavoured controller.